### PR TITLE
Merge 40 to main

### PR DESCRIPTION
**Pull request by Callisto13**
The auto job failed.

### CODE REVIEW

File: pkg/version/release.go [hunk 1]
```diff
@@ -6,7 +6,7 @@ package version
 var Version = "0.40.0"
 
 // PreReleaseID can be empty for releases, "rc.X" for release candidates and "dev" for snapshots
-var PreReleaseID = "dev"
+var PreReleaseID = "rc.0"
```
Comment (aclevername) [inline pkg/version/release.go:9]: we don't want this in main right?

Comment (Callisto13) [reply pkg/version/release.go:9]: honest to god no idea 🤷‍♀️

Comment (Callisto13) [reply pkg/version/release.go:9]: and some merge to main is failing below

Comment (aclevername) [reply pkg/version/release.go:9]: we definitely don't want this in main as we want to keep cutting dev releases by default

**Pull request closed without merging**